packages/splitter/splitter.stories.tsx

```diff
@@ -89,10 +92,10 @@ export const Container = ({
               }}
             >
               <div
-                id="primary-pane"
+                {...primaryPaneProps}
                 style={{
-                  ...primaryPaneStyle,
-                  flexBasis: `${separatorProps['aria-valuenow']}px`
+                  ...primaryPaneProps.style,
+                  flexBasis: primaryPaneProps.valueNow
```

Comment:
Good Question.

The `aria-valuenow` is delivered to separator props to inform screen readers the current state of the Separator position as required by WAI ARIA.

The resize of content is completely reliant on the sizing of the pane. Therefore if `getPrimaryPaneProps` is solely responsible for _all required properties_ intended for basic functionality then we would need to deliver the sizing value manage the primary pane. 

I chose `valueNow` over `aria-valuenow` in primary pane as I wanted to force the developer along the path to use this in React layout logic rather than deliver an ARIA attribute to an HTML element that shouldn't have it. 

It seems rather indirect to have properties that overlap in concerns between separator and primary pane. So Im unconvinced to change this.

Reply:
The current HTML markup for the primary pane element fails https://validator.w3.org and React is logging a warning because of it.

Reply:
Ill destructure `valueNow` off of the `div`. 👍

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/f2ee30a39269af3af40b9dc1c59a6b20de5edd03